data/sheragi/archaeology missions.txt

```diff
@@ -895,7 +899,9 @@ mission "Mammon Payment"
 
 	to offer
 		random < 10
-		has "Sheragi Archaeology: Emerald Sword Epilogue: done"
+		or
+			has "Sheragi Archaeology: Emerald Sword Epilogue: done"
+			has "event: mammon impatient"
```

Comment:
Previously, Mammon would only ever take credits from the player after they finished the Emerald Sword missions. But if the player never finishes the Emerald Sword missions (perhaps they aborted/failed a mission) then Mammon never gets paid. Added an alternative condition where Mammon might take credits from you 240 days after having given you info on where the cube is, ensuring that Mammon gets paid no matter what the player ends up doing.

Reply:
Awesome, i was trying to figure out a solution for that, didn't remember about the or condition.